index.rst

```diff
@@ -38,11 +40,33 @@ of text editor plugins that enable editors to read the file format and adhere
 to defined styles. EditorConfig files are easily readable and they work nicely
 with version control systems.
 
+
+Terminology
+===========
+
+In EditorConfig:
+
+- "EditorConfig files" (usually named ``.editorconfig``) store settings,
+  and must conform to this specification.
+- "Cores" parse files conforming to this specification.
+- "Plugins" apply settings to files being edited, and use cores to
+  determine the settings.
+- "Editors" permit editing files, and use plugins to apply settings.
+
+A conforming core or plugin must pass the tests in the
+`core-tests repository`_ or `plugin-tests repository`_, respectively.
+
+*(informative)* Some plugins include or bundle their own cores, and some rely
+on external cores.  Some editors include or bundle plugin or core
+functionality.  Editors, plugins, and cores may all come from different
+people.  Those people may or may not have any direct interaction with the
+EditorConfig organization.
+
 File Format
 ===========
 
```

Comment:
Nitpicky but this change does not deal with versioning. It's a great call out that it's INI like but that warrants a separate PR and maybe another paragraph in this file.

Reply:
I think you are referring to old line 44 / new line 68.  Yes?  I did indeed make some cleanup changes throughout in this PR along with the versioning changes.  I certainly can move them to a separate PR but would prefer not to since they are so small.

The "INI-like" nature has been discussed many times, e.g., https://github.com/editorconfig/editorconfig/issues/279 .

Reply:
Can live with that, as said very nitpicky. Happy to see this go in.

Reply:
Much appreciated!